aikau/src/main/resources/alfresco/core/Page.js

```diff
@@ -39,10 +41,10 @@ define(["alfresco/core/ProcessWidgets",
         "jquery", // NOTE: Need to include JQuery at root page to prevent XHR require request for first module that uses it
         "jqueryui", // NOTE: Need to include JQuery UI at root page to prevent XHR require request for first module that uses it
         "alfresco/core/shims"],
-        function(ProcessWidgets, AlfConstants, string, declare, domConstruct, array,
+        function(ProcessWidgets, TopicsMixin, AlfConstants, string, declare, domConstruct, array,
                  lang, domClass, win, PubQueue, jquery, jqueryui, shims) {
 
-   return declare([ProcessWidgets], {
+   return declare([ProcessWidgets, TopicsMixin], {
```

Comment:
Argh! We shouldn't - that's what we discussed..... Core should not mixin TopicsMixin - not all widgets need access to it - I obviously missed this on a previous code review and it should be changed.